src/fns/unconstrained_ops.nr

```diff
@@ -171,34 +171,38 @@ pub(crate) unconstrained fn __udiv_mod<let N: u32>(
     let mut divisor_u60: U60Repr<N, 2> = U60Repr::from(divisor);
     let b = divisor_u60;
 
-    let mut bit_difference = remainder_u60.get_msb() - divisor_u60.get_msb();
+    if !remainder_u60.gte(divisor_u60) {
```

Comment:
I wonder if there would be a better way to catch this, without using the if statement. @TomAFrench would it be more efficient to look at the index where the `msb` of the denominator is rather than using the if statement? 
regardless the change should fix this issue.

Reply:
@Pablo-Dallegri would you like to tackle this yourself or should I try to make the change before merging the PR?